Android/app/src/main/java/tie/hackathon/travelguide/destinations/description/FinalCityInfo.java

```diff
@@ -17,24 +16,17 @@
 import com.ms.square.android.expandabletextview.ExpandableTextView;
 import com.squareup.picasso.Picasso;
 
-import org.json.JSONException;
-import org.json.JSONObject;
-
-import java.io.IOException;
-
-import Util.Constants;
 import butterknife.BindView;
 import butterknife.ButterKnife;
-import okhttp3.Call;
-import okhttp3.Callback;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
+import tie.hackathon.travelguide.FunFacts;
+import tie.hackathon.travelguide.PlacesOnMap;
+import tie.hackathon.travelguide.R;
+import tie.hackathon.travelguide.Tweets;
 
 /**
  * Fetch city information for given city id
  */
-public class FinalCityInfo extends AppCompatActivity implements View.OnClickListener {
+public class FinalCityInfo extends AppCompatActivity implements View.OnClickListener, FinalCityInfoView {
 
     private Intent intent;
     private TextView fftext;
```

Comment:
These were already present, it definitely hit my mind to change these names but wanted to re-use old code as much as possible. I'll change them.